packages/core/src/Modifiers.ts

```diff
@@ -1,6 +1,12 @@
 import { Modifier } from './Modifier';
 import { Constructor, isConstructor } from '../../utils/src/utils';
 
+/**
+ * When parsing dom elements, modifiers can be DOM inline nodes (a, b, strong).
```

Comment:
Don't write this here.
1. No mention of the DOM in `core`
2. No mention of a plugin in `core` (`Format` in this case)
3. Start your description of the class by talking about the general case, the general purpose, not the edge cases.
4. Be consistent: you write "dom" and "DOM" in the same sentence.